include/syntax/c

```diff
@@ -441,3 +441,5 @@ code:yylessthan =
 code:yybm_filter = yych " & ~0xFF";
 
 code:yybm_match = yybm "[" offset "+" yych "] & " mask;
+
+code:yytarget_elem = (.cgoto.relative ? "(int)((char*)&&" : "&&") label (.cgoto.relative ? " - (char*)&&" base ")");
```

Comment:
I think it will be a bit cleaner if you totally split the two cases, like this:
```
code:yytarget_elem = (.cgoto.relative
    ? "(int)((char*)&&" label " - (char*)&&" base ")"
    : "&&" label);
```